.run/build.sh

```diff
@@ -0,0 +1,11 @@
+ROOT_DIR=/git/datadog-agent
```

Comment:
#! /usr/bin/bash -ex

Reply:
the script can be run on bare linux machine and thus the root dir does not match everyone's local repo

You can use `$(dirname $(dirname $(realpath "$0")))` or `$(git rev-parse --show-toplevel)`

Reply:
Used `$(git rev-parse --show-toplevel)` but also had to add a cd command for the ssh configuration so it will work